examples/keyvalue/Makefile

```diff
@@ -7,4 +9,7 @@ server/server: $(wildcard server/*.go) kv/types.go
 .PHONY: all
 all: server/server
 
-.DEFAULT_GOAL: all
+.PHONY: deps
+deps:
+	$(ECHO_V)go get -v go.uber.org/thriftrw
```

Comment:
Wait. I though we were installing from ../../vendor/ and not do go get? This will lead to failures once yarpc update happens.